packages/web/src/components/modal-radio/ModalRadioItem.module.css

```diff
@@ -52,3 +52,27 @@
 .collapsibleContainer.collapsed {
   display: none;
 }
+
+.hint {
+  display: flex;
+  gap: var(--unit-4);
+  align-items: center;
+  padding: var(--unit-3) var(--unit-4);
+  background: var(--background-surface-1);
+  border: 1px solid var(--border-strong);
+  border-radius: var(--unit-2);
```

Comment:
can we use https://github.com/AudiusProject/audius-client/blob/main/packages/web/src/components/help-callout/HelpCallout.tsx#L10 for this?

Reply:
wow, i looked all over for this. thank you lol

Reply:
lmao yeah I found it in the old `TrackAvailabilityModal`